scio-core/src/main/scala/com/spotify/scio/util/ScioUtil.scala

```diff
@@ -18,23 +18,28 @@
 package com.spotify.scio.util
 
 import java.net.URI
+import java.util.UUID
 
 import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import com.google.api.client.googleapis.services.AbstractGoogleClientRequest
 import com.google.api.services.dataflow.Dataflow
 import com.google.api.services.dataflow.model.JobMetrics
+import com.spotify.scio.ScioContext
 import org.apache.beam.runners.dataflow.options._
 import org.apache.beam.runners.direct.DirectRunner
 import org.apache.beam.sdk.coders.{Coder, CoderRegistry}
 import org.apache.beam.sdk.io.gcp.bigquery.BigQueryTableRowIterator
-import org.apache.beam.sdk.options.PipelineOptions
+import org.apache.beam.sdk.options.{GcpOptions, PipelineOptions}
 import org.apache.beam.sdk.util.Transport
+import org.slf4j.LoggerFactory
 
 import scala.reflect.ClassTag
+import scala.util.{Failure, Success}
 
 private[scio] object ScioUtil {
 
+  @transient lazy private val log = LoggerFactory.getLogger(this.getClass)
```

Comment:
`private lazy val` so it's easy to grep for `lazy val`?
~There's one in `SCollection` as well but we can fix that separately.~ Just realized this PR removed that one.